src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -334,8 +334,7 @@ public static string GetName(Type eventSourceType)
         }
 
 #if !ES_BUILD_STANDALONE
-        private const DynamicallyAccessedMemberTypes ManifestMemberTypes = DynamicallyAccessedMemberTypes.PublicNestedTypes
-            | DynamicallyAccessedMemberTypes.PublicMethods | DynamicallyAccessedMemberTypes.NonPublicMethods;
+        private const DynamicallyAccessedMemberTypes ManifestMemberTypes = DynamicallyAccessedMemberTypes.All;
```

Comment:
We wouldn't need to bump this to "keep the entire type" if we did https://github.com/mono/linker/pull/1719#issuecomment-753444772

Reply:
I took the suppression attribute out since that is how I understood Vitek's fix to be. Let me know if the fix will also allow DynamicallyAccessedMemberTypes.All annotation to be replaced with an appropriate subset (my test didn't show that to be the case)